entrypoint.sh

```diff
@@ -18,6 +18,16 @@ sh -c "git config --global credential.username $GITLAB_USERNAME"
 sh -c "git config --global core.askPass /cred-helper.sh"
 sh -c "git config --global credential.helper cache"
 sh -c "git remote add mirror $*"
+if [[ -n "${REMOVE_BRANCH}" ]]; then # Check if variable exists 
+   if [[ "${REMOVE_BRANCH}" == "true" ]]; then 
+      # If branch exists; w/o sh -c ""
+      branchExists=$(git ls-remote $(git remote get-url --push mirror) ${CHECKOUT_BRANCH:-$DEFAULT_GITHUB_REF} | wc -l)
+      if [[ ${branchExists} == 1 ]]; then
```

Comment:
In bash, isn't it better to quote both the variable size and the comparison side of a binary operator?

Reply:
I guess. I will change it